pkg/flag/flag.go

```diff
@@ -28,23 +28,34 @@ const (
 	TriggerControllerName = "tekton-triggers-controller"
 	TriggerWebhookName    = "tekton-triggers-webhook"
 
-	AnnotationPreserveNS = "operator.tekton.dev/preserve-namespace"
+	AnnotationPreserveNS      = "operator.tekton.dev/preserve-namespace"
+	LabelTaskProviderType     = "operator.tekton.dev/provider-type"
+	TaskProviderTypeCommunity = "community"
+	TaskProviderTypeRedHat    = "redhat"
+	TaskProviderTypeCertified = "certified"
 
 	uuidPath = "deploy/uuid"
 )
 
 var (
 	flagSet *pflag.FlagSet
 
-	TektonVersion   = "v0.9.2"
-	PipelineSA      string
-	IgnorePattern   string
-	ResourceWatched string
-	ResourceDir     string
-	TargetNamespace string
-	NoAutoInstall   bool
-	Recursive       bool
-	OperatorUUID    string
+	TektonVersion          = "v0.9.2"
+	PipelineSA             string
+	IgnorePattern          string
+	ResourceWatched        string
+	ResourceDir            string
+	TargetNamespace        string
+	NoAutoInstall          bool
+	SkipNonRedHatResources bool
+	Recursive              bool
+	OperatorUUID           string
+	NonRedHatResourceURLs  = []string{
```

Comment:
how about using a config-map for this?

Reply:
we can technically do it, but it will change the order of manifest creation.
First the config-map will have to be created as part of `addons`.
Then, in the next phase we will have to read the configMap, download tasks and apply them.

In the current scope, as we are not providing reconcile capability if the contents of configMap changes, IMHO i think it will be an overkill.

@sthaha @hrishin